thirdparty/basis_universal/encoder/basisu_comp.cpp

```diff
@@ -4071,7 +4071,7 @@ namespace basisu
 				comp_params.m_source_mipmap_images_hdr.resize(1);
 				comp_params.m_source_mipmap_images_hdr[0].resize(pSource_images_hdr->size() - 1);
 
-				for (uint32_t i = 1; i < pSource_images->size(); i++)
+				for (uint32_t i = 1; i < pSource_images_hdr->size(); i++)
```

Comment:
This looks correct, since this branch uses `_hdr`, and only called when `pSource_images` is null.

Changes in third-party code should have `patch` file for the change (see other third-party libs for examples), and it should be mentioned in the `thirdparty/README.md`. Also consider submitting it to upstream - https://github.com/BinomialLLC/basis_universal.